frontend/src/metabase/query_builder/components/filters/modals/BulkFilterSelect/BulkFilterSelect.styled.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from "@emotion/styled";
+import SelectButton from "metabase/core/components/SelectButton";
+import FilterPopover from "../../FilterPopover";
+
+export const SelectFilterButton = styled(SelectButton)`
+  min-height: 2.25rem;
+
+  &:not(:first-child) {
```

Comment:
Same question about using `first-of-type` instead because of emotion noise

Reply:
Fixed